lib/presets/safe.es6

```diff
@@ -26,6 +26,7 @@ export default {
     removeEmptyAttributes: true,
     removeRedundantAttributes: false,
     removeComments: 'safe',
+    removeAttributeQuotes: false,
```

Comment:
Although the feature is safe to enable (won't break anything), but it requires a completely rewritten for every existing unit test cases.

Since `html-minifier` doesn't enable the feature by default, I make it disabled in `safe` presets as well.